_data/browsers.yml

```diff
@@ -1,17 +1,23 @@
 - firefox:
   vendor: Mozilla
   name: Firefox
+  version: 77
 
 - chrome:
   vendor: Google
   name: Chrome
+  version: 85
 
 - edge:
   vendor: Microsoft
   name: Edge
+  version: 85
 
 - safari:
   vendor: Apple
   os: iOS
   name: Safari
-  version: 10.3
\ No newline at end of file
+  version: 13.4
+
+- opera:
+  name: Opera
```

Comment:
Whoops! I didn't mean to push up this old part of the file. I'll remove in a force push.